test/OpenTelemetry.Tests/Instrumentation/PropertyFetcherTest.cs

```diff
@@ -37,10 +36,10 @@ public void FetchInvalidProperty()
         {
             var activity = new Activity("test");
             var fetch = new PropertyFetcher<string>("DisplayName2");
-            var result = fetch.Fetch(activity);
+            Assert.False(fetch.TryFetch(activity, out string result));
```

Comment:
Would you add a test case for the null (eg: `fetch.TryFetch(null, out string result)`)? I forgot to do that in my branch sorry!

Reply:
done